use-cases/virgo/config.yaml

```diff
@@ -1,6 +1,9 @@
 root_folder: /p/scratch/intertwin/datasets/virgo
+hdf5_file_location: /p/project1/intertwin/saether1/itwinai/use-cases/virgo/data/virgo_data.hdf5
+hdf5_dataset_name: virgo_dataset
+chunk_size: 5
```

Comment:
to avoid confusion, you could force the batch_size in the DataLoader to be always 1, allowing to control the actual batch size by means of the chunk_size. AFAIU, now the *real* batch size is equal to `batch_size  * chunk_size`. If this is the case, I would rename `chunk_size` in the top-level configuration as batch_size and remove batch_size. Down in the config I would then specify   

```yaml
          hdf5_file_location: ${hdf5_file_location}
          chunk_size: ${batch_size}
          hdf5_dataset_name: ${hdf5_dataset_name}
```

and where before we had  `batch_size: ${batch_size}` I would replace it with `batch_size: 1`.

See comments below